src/package.json

```diff
@@ -55,6 +55,26 @@
     "bootstrap": "^3.3.7",
     "d3": "^5.9.2"
   },
+  "exports": {
+    "./styles/ux-aspects.css": {
+      "style": "./styles/ux-aspects.css"
+    },
+    "./styles/ux-aspects.min.css": {
+      "style": "./styles/ux-aspects.min.css"
+    },
+    "./less/ux-aspects.less": {
+      "sass": "./less/ux-aspects.less"
```

Comment:
I don't believe this would be `sass` as that suggests it would be imported from a sass stylesheet, instead I believe this should be `style` since there is no `less` conditional.